config/workbox-cli-config.js

```diff
@@ -0,0 +1,36 @@
+const paths = require('./paths.json') // specify paths to main working directories
+
+module.exports = {
+  globDirectory: paths.public,
+  globPatterns: [
+    '**/*.{html,js,css,png,jpg}'
+  ],
+  swDest: paths.public + 'service-worker.js',
+  importWorkboxFrom: 'local',
+  skipWaiting: true,
+  clientsClaim: true,
+  directoryIndex: 'index.html',
+  cacheId: 'govuk-ds',
+  runtimeCaching: [{
+    urlPattern: /.*/,
+    handler: 'networkFirst'
+  },
+  {
+    urlPattern: new RegExp('https://www.google-analytics.com/analytics.js'),
+    handler: 'networkFirst',
+    options: {
+      expiration: {
+        maxAgeSeconds: 60 * 60
+      }
+    }
+  },
+  {
+    urlPattern: new RegExp('https://www.googletagmanager.com/gtm.js'),
```

Comment:
Wondering if we could do `https:\/\/www\.google.*\.com\/.*\.js` here instead, which matches any javascript file from a domain that starts with google?

Reply:
Sure, we can do that. I was trying to be explicit as I'm unsure what other Google URL's we may be using in the future.

Reply:
Let's go with this for now then. 👍